src/security/data-deletion.md

```diff
@@ -0,0 +1,17 @@
+# Data Deletion
+
+## Data deletion
```

Comment:
No need to have double headers here.  Just remove the second, and use "Sentence case" for the title header.